sorc/ncep_post.fd/WRFPOST.f

```diff
@@ -843,18 +843,17 @@ PROGRAM WRFPOST
           PRINT*,'UNKNOWN MODEL OUTPUT FORMAT, STOPPING'
           STOP 9999
         END IF 
-        INITPOST_tim  = INITPOST_tim +(timef() - btim)
-        time_initpost = time_initpost + timef()
+        INITPOST_tim  = INITPOST_tim +(mpi_wtime() - btim)
         IF(ME == 0)THEN
           WRITE(6,*)'WRFPOST:  INITIALIZED POST COMMON BLOCKS'
         ENDIF
 !
 !       IF GRIB2 read out post aviable fields xml file and post control file
 !
         if(grib == "grib2") then
-          btim=timef()
+          btim=mpi_wtime()
           call READ_xml()
-          READxml_tim = READxml_tim + (timef() - btim)
+          READxml_tim = READxml_tim + (mpi_wtime() - btim)
```

Comment:
@JesseMeng-NOAA Please update the timers in other routines e.g. PROCESS.f with Jim's changes.